dist/index.html

```diff
@@ -0,0 +1,60 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./style.css" />
+    <title>To-Do Lists with Js</title>
+  </head>
+  <body>
+    <header>
+      <h1>To-Do List</h1>
+      <div class="header-selectors">
+        <p>Show:</p>
+        <select id="projects-view-selector" name="projects" disabled>
+          <option value="default">All</option>
+        </select>
+        <p>Order by:</p>
+        <select id="orderBy" name="orderBy" disabled>
+          <option class="option-list" value="order-by-priority">
+            Priority
+          </option>
+          <option class="option-list" value="order-by-duedate">Due Date</option>
+        </select>
+        <p>New Project:</p>
+        <input class="input-project-name" type="text" />
+        <button class="new-project-submit">+</button>
+      </div>
+    </header>
+
+    <div class="new-task-txt-input-div">
+      <input
+        placeholder="Add your new task here..."
+        type="text"
+        id="input-todo-name"
+      />
+      <select id="select-project"></select>
```

Comment:
## Remove unnecessary fields
- [ ] Kindly remove unnecessary busy elements on the page to make it less busy. 
- [ ] The `add new task` below the project is unnecessary as the functionality has been implemented in the form up above. Kindly remove it.
- [ ] The `show` and `order by` fields in the top need also to be removed for the same reason.
![Screenshot 2021-02-06 115510](https://user-images.githubusercontent.com/22345961/107113971-8b264180-6873-11eb-9a6c-1b72275bc616.png)